recipes-bsp/dp-firmware-cadence/dp-firmware-cadence_20.04.bb

```diff
@@ -1,15 +1,15 @@
 SUMMARY = "DP firmware"
 LICENSE = "Proprietary"
-LIC_FILES_CHKSUM = "file://COPYING;md5=6c12031a11b81db21cdfe0be88cac4b3"
+LIC_FILES_CHKSUM = "file://COPYING;md5=228c72f2a91452b8a03c4cab30f30ef9"
```

Comment:
Why the license checksum must be changed?

http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines

Reply:
update commit message.